lib/tools/adb-commands.js

```diff
@@ -727,6 +734,32 @@ methods.setWifiAndData = async function ({wifi, data}, isEmulator = false) {
 };
 
 /**
+ * Change the state of animation on the device under test.
+ *
+ * @param {boolean} on - True to enable and false to disable it.
+ */
+methods.setAnimationState = async function (on) {
+  await this.shell(['am', 'broadcast', '-a', `${ANIMATION_SETTING_ACTION}`,
+                    '-n', `${SETTINGS_HELPER_ID}/${ANIMATION_SETTING_RECEIVER}`,
+                    '--es', 'setstatus', on ? 'enable' : 'disable']);
+};
+
+/**
+ * Check the state of animation on the device under test.
+ *
+ * @return {boolean} True if at least one of animation scale settings
+ *                   is not equal to zero.
+ */
+methods.isAnimationOn = async function () {
+  let animator_duration_scale = await this.getSetting('global', 'animator_duration_scale');
+  let transition_animation_scale = await this.getSetting('global', 'transition_animation_scale');
+  let window_animation_scale = await this.getSetting('global', 'window_animation_scale');
+  return (parseFloat(animator_duration_scale, 10) !== 0 ||
```

Comment:
you cannot compare float values this way. Either cast it to integer or use [Number.ESPILON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON) constant